corgi/collectors/rhel_compose.py

```diff
@@ -2,22 +2,73 @@
 import os
 from collections import defaultdict
 from datetime import datetime
-from typing import Tuple
+from typing import Any, Generator, Tuple
 
 import requests
 
 from corgi.collectors.brew import Brew
+from corgi.collectors.models import (
+    CollectorComposeRhelModule,
+    CollectorComposeRPM,
+    CollectorComposeSRPM,
+)
 
 logger = logging.getLogger(__name__)
 
 
 class RhelCompose:
+    @classmethod
+    def fetch_rhel_module(cls, build_id: int) -> dict[str, Any]:
+        try:
+            rhel_module = CollectorComposeRhelModule.objects.get(build_id=build_id)
+        except CollectorComposeRhelModule.DoesNotExist:
+            logger.debug("Did not find %s in CollectorComposeRhelModule data", build_id)
```

Comment:
this try existing probably might mean switching logger.debug to logger.info

Reply:
It's expected behavior that CCRhelModule is not exists in many cases. That's because we also have many non-modular SRPMs in COMPOSE relations as well. All of those will log something, which is why I set it to DEBUG, because we probably don't care 99% of the time.